test/ssl_test_ctx.c

```diff
@@ -286,6 +286,29 @@ const char *ssl_session_ticket_name(ssl_session_ticket_t server)
                      server);
 }
 
+/* CompressionExpected */
+
+static const test_enum ssl_compression[] = {
```

Comment:
why do we have separate types for all these booleans?  why not just a test_on test_off ? 
okay, it's not "all" just one other one.  but it should be merged.